tests/pytorch/test_pytorch_model_export.py

```diff
@@ -807,6 +808,11 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
 
 
 @pytest.mark.large
+@pytest.mark.skipif(
+    Version(torch.__version__) >= Version("1.9.0"),
+    reason="From pytorch 1.9.0, torch.load() creates a subclass from the passed Unpickler. "
+           "Refer https://github.com/pytorch/pytorch/pull/53139",
+)
```

Comment:
In pytorch 1.9.0, a subclass is created from the passed Unpickler in the torch.load function as pickle.Unpickler is marked readonly in pickle.
For more details refer to [this PR](https://github.com/pytorch/pytorch/pull/53139)